app/src/main/java/com/cornellappdev/scoop/ui/components/post/SecondPage.kt

```diff
@@ -282,48 +289,33 @@ fun TimeOfTripSection(
 
     Column(
         modifier = Modifier
-            .width(200.dp)
             .padding(top = 30.dp)
     ) {
-        Text(
-            text = stringResource(R.string.time_of_trip),
-            fontSize = 22.sp,
-            modifier = Modifier.padding(bottom = 12.dp)
-        )
-        Row {
+        // Box stacking a clickable Clock icon on top of a DenseTextField
+        Box (modifier = Modifier.fillMaxWidth()) {
+            Box(Modifier.fillMaxWidth()) {
```

Comment:
delete the second Box(Modifier.fillMaxWidth()) since it's redundant with the first one